langchain4j-core/src/main/java/dev/langchain4j/audit/api/event/LLMInteractionCompleteEvent.java

```diff
@@ -0,0 +1,82 @@
+package dev.langchain4j.audit.api.event;
+
+import dev.langchain4j.audit.event.DefaultLLMInteractionCompleteEvent;
+import java.util.Optional;
+import org.jspecify.annotations.Nullable;
+
+/**
+ * Represents an event that occurs upon the completion of an interaction
+ * with a large language model (LLM). This interface extends {@link LLMInteractionEvent}
+ * to include additional information about the result of the interaction.
+ *
+ * Classes implementing this interface are expected to provide details
+ * on what constitutes the result of an interaction, which may vary based
+ * on the specifics of the use case. The result can represent the outcome
+ * of the interaction process, a returned value, a processed value, or
+ * {@code null} if no meaningful result exists.
+ */
+public interface LLMInteractionCompleteEvent extends LLMInteractionEvent {
```

Comment:
```suggestion
public interface LLMInteractionCompletedEvent extends LLMInteractionEvent {
```
to keep consistent with other `*edEvent` classes

Reply:
But in the end I would name it `AiServiceInvocationCompletedEvent`

Reply:
Done